lib/tasks/simulate.rake

```diff
@@ -1,18 +1,22 @@
 namespace :simulate do
   desc "Simulates opening a new pull request on github"
-  task pull_request: :environment do
-    post_github_event('pull_request', render_new_pull_request_json)
+  task :pull_request, [:count] => :environment do |_t, args|
+    args.with_defaults(count: 1)
+    disable_github_authentication
+    args[:count].to_i.times { post_github_event('pull_request', render_new_pull_request_json) }
   end
 
   desc "Simulates a new commit on github"
   task :commit, [:count] => :environment do |_t, args|
     args.with_defaults(count: 1)
+    disable_github_authentication
```

Comment:
Would this get its own commit?

Reply:
Hrm. Maybe. The issue was calling `disable_github_authentication` multiple times was throwing any error since it can't remove a controller lifecycle callback more than once. That is only a bug if you want to `post_github_event` more than once.

Reply:
Yeah, ok. I'm sold after typing that all out.

Reply:
Interesting -- I'dve been happy either way so 💯 🏅